web_poet/page_inputs.py

```diff
@@ -15,3 +15,38 @@ class ResponseData:
     """
     url: str
     html: str
+
+
+@attr.s(auto_attribs=True)
+class SplashResponseData:
+    """A container for a response, rendered using Splash browser.
+
+    ``url`` should be an URL of the response (after all redirects),
+    not an URL of the request, if possible.
+
+    ``html`` should be content of the HTTP body, converted to unicode
+    using the detected encoding of the response, preferably according
+    to the web browser rules (respecting Content-Type header, etc.)
+    """
+    url: str
+    sourceUrl: str
+    html: str
+    jpeg: bytes
+    httpStatus: int
+
+
+@attr.s(auto_attribs=True)
+class AutoextractProductResponse:
```

Comment:
It would be awesome if you could mention something here in a docstring (maybe linking to AutoExtract docs).